tests/churn/churn.cpp

```diff
@@ -161,7 +165,7 @@ class pageiotest {
           for (uint64_t i = 0; i < num_churn_pages * (pagesize/sizeof(*p)); i += (pagesize/sizeof(*p)))
               if (p[i] != i) {
                 cerr << "check(CHURN): *(uint64_t*)" << &p[i] << "=" << p[i] << " != " << (unsigned long long)i << endl;
-                return;
+                exit(1);
```

Comment:
This will exit after the first error, so subsequent errors won't be reported. Same for change on line 177. Is this the intended behavior?

Reply:
Yes:-).  Although we could arguably make another option for that if we wanted to.  What I have seen is that when data miss-compares occur, LOTS of them occur.  It is usually enough to stop when we see data corruption that we should not.

Reply:
That may not be necessary for churn. When I was debugging the sort benchmark, it was useful to see multiple wrong values. However, I agree that having thousands of them get printed in churn is not useful.